Header header

float64[3] hand_direction
float64[3] palm_normal
float64[3] palm_position
float64[3] ypr
float64[3] thumb_tip
float64[3] index_tip
float64[3] middle_tip
float64[3] ring_tip
float64[3] pinky_tip
float64[5] finger_angles
string mode